docs/reST/ref/key.rst

```diff
@@ -269,6 +269,40 @@ for ``KMOD_NONE``, which should be compared using equals ``==``). For example:
 
    .. ## pygame.key.get_pressed ##
 
+.. function:: get_just_pressed
+
+   | :sl:`returns a pygame.key.ScancodeWrapper containing the most recent key presses`
+   | :sg:`get_just_pressed() -> bools`
+
+   Returns a sequence of boolean values representing the most recent key presses.
+   When the event loop is pumped using pygame.event.get() If the event is found to be an
+   SDL_KEYDOWN the SDL_Event.key.keysym.scancode is used to update the most recent key presses.
```

Comment:
I wrote a long review with suggestions about how to reword this and things, but I apparently never posted it?

Things
- It should say that it tracks the KEYDOWNs behind the scenes when the event system is pumped (for example by pygame.event.get())
- Therefore it's similar to checking KEYDOWN in the event loop
- It should say, as a note (.. note::), that if you require getting presses in order, you should still use the event queue KEYDOWN events.
- The fact that it's an SDL_KEYDOWN or that the scancode is what's being used to address into things is not relevant to the reader